doc/oemof_solph.rst

```diff
@@ -46,17 +46,16 @@ Now you can start to add the components of the network.
 Add your components to the energy system
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you have defined an instance of the EnergySystem class all components you define will automatically be added to your EnergySystem.
+After defining an instance of the EnergySystem class you have to add all components you define in the following to your EnergySystem.
```

Comment:
Components is right in trivial language but in our case we have to say nodes. Nodes can be buses or components and components can be sources, sinks,.... see https://github.com/oemof/oemof/blob/dev/oemof/network.py#L248